Proto/cg.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
I don't know if people were expecting it, but I'll reinstate (not being the first one) that using FAKE probably makes the most sense, even if it is just for the top level targets/dependency system as well as set of msbuild helpers (+ those bits that will come for .net core support), the commitment to it will necessarily grow over time even if initial use was superficial.

This is what a first shot at reverse-engineering the main build.bat (for the part allowing me to build proto and net40) looked like https://github.com/smoothdeveloper/buildvisualfsharp/blob/master/fakebuild.fsx

This brings the ability to step/evaluate each build step, inspect variables etc. An invaluable tool.

Raw F# script also works (as @enricosada explains well), but why write so much of low level infrastructure stuff when the helpers in FAKE are already available and well understood by F# community / FAKE users? Eventually if we want to tailor / optimize parts of our scripts, we can still do so, I'm sure there are lots of place where it has added value to have this degree of control too.

Regarding concern of needing dependency on Mono, I think it is not a big deal (mono is already in every package manager on macosx and linux) and also we put additional motivation for projects like FAKE (and who knows paket?) to support .net Core ASAP to have less dependencies to build F# on other platforms than windows.

Assuming FakeLib compiles for .net core we can use it with plain fsi.exe without depending on mono so it is something which can be figured out, I think right now anyways, this repository is not able to get built on other platforms than windows, switching to F# scripting is a significant step in that direction.